Gems/AtomLyIntegration/gem.json

```diff
@@ -25,5 +25,16 @@
         "CommonFeaturesAtom",
         "EMotionFX_Atom",
         "ImguiAtom"
+    ],
+    "external_subdirectories": [
+        "AtomBridge",
+        "AtomFont",
+        "AtomImGuiTools",
+        "AtomViewportDisplayIcons",
+        "AtomViewportDisplayInfo",
+        "CommonFeatures",
+        "EmotionFXAtom",
+        "ImGuiAtom",
+        "TechnicalArt"
     ]
```

Comment:
We should wait on adding an "external_subdirectories" field here.
I see this change doesn't add any CMake support for invoking add_subdirectory on those directories and this is not gaining us anything at the moment.